dashboard/src/actions/keyManagementActions.js

```diff
@@ -0,0 +1,98 @@
+import * as TYPES from "actions/types";
+
+import { DANGER, ERROR_MSG, SUCCESS } from "assets/constants/toastConstants";
+
+import API from "../utils/axiosInstance";
+import { showToast } from "./toastActions";
+import { uriTemplate } from "utils/helper";
+
+export const getAPIkeysList = () => async (dispatch, getState) => {
+  try {
+    dispatch({ type: TYPES.LOADING });
+
+    const endpoints = getState().apiEndpoint.endpoints;
+    const response = await API.get(uriTemplate(endpoints, "key", { key: "" }));
+
+    if (response.status === 200) {
+      dispatch({
+        type: TYPES.SET_API_KEY_LIST,
+        payload: response.data,
+      });
+    } else {
+      dispatch(showToast(DANGER, ERROR_MSG));
+    }
+  } catch (error) {
+    dispatch(showToast(DANGER, error));
+  }
+  dispatch({ type: TYPES.COMPLETED });
+};
+
+export const triggerDeleteAPIKey = (id) => async (dispatch, getState) => {
```

Comment:
Not a serious problem, but why `triggerDeleteAPIKey` instead of just `deleteAPIKey`?

Reply:
Updated